components/util/socialIcons.tsx

```diff
@@ -103,7 +103,18 @@ export interface SocialIconsParams {
   excludeMobile?: SocialTypes[];
 }
 
+export const isMobileDeviceAgent = () => {
+  return /Mobile|iP(hone|od|ad)|Android|BlackBerry|IEMobile/.test(
+    window.navigator.userAgent
+  );
+};
+
```

Comment:
Can you move this to a file in /services - so its easier for devs to see they can use it

Reply:
I just replaced this function with a library named `react-device-detect` , as here we were using regex to test and it is not reliable as well as testable.